include/nbl/asset/utils/CDirQuantCacheBase.h

```diff
@@ -43,25 +43,26 @@ class CDirQuantCacheBase
 				
 				Vector8u3() : x(0u),y(0u),z(0u) {}
 				Vector8u3(const Vector8u3&) = default;
-				explicit Vector8u3(const core::vectorSIMDu32& val)
+				explicit Vector8u3(const hlsl::float32_t3& val)
 				{
 					operator=(val);
 				}
 
 				Vector8u3& operator=(const Vector8u3&) = default;
-				Vector8u3& operator=(const core::vectorSIMDu32& val)
+				Vector8u3& operator=(const hlsl::float32_t3& val)
 				{
 					x = val.x;
 					y = val.y;
 					z = val.z;
 					return *this;
 				}
 
-				inline core::vectorSIMDu32 getValue() const
+        hlsl::float32_t3 getValue() const
 				{
-					return core::vectorSIMDu32(x,y,z);
```

Comment:
this used to return the counterpart to `uint32_t4`, it could return `vector<uint8_t,3>` I guess, but definitely not float